modules/endpoint-group/README.md

```diff
@@ -2,6 +2,36 @@
 
 Submodule for provisioning Global Accelerator Endpoint Groups.
 
+This submodule takes in a variable `config` that is essentially fully compliant with the `aws_globalaccelerator_endpoint_group` resource, except for the following differences:
+
+* `listener_arn`, which is specified separately, is omitted.
+* The values for `endpoint_configuration` and `port_override` within each object in `endpoint_groups` should be lists.
+* Inside the `endpoint_configuration` block, `endpoint_lb_name` can be supplied in place of `endpoint_id` as long as it is a valid unique name for an existing ALB or NLB.
+* Inside the `endpoint_configuration` block, `endpoint_eip_address` can be supplied in place of `endpoint_id` as long as it is a valid public IP address for an Elastic IP.
```

Comment:
This is all duplicated in the variable definition which gets pulled in automatically into the `README` so couldn't this block be omitted ?

Reply:
It is good to highlight information like this. I approve.

Reply:
Resolved